cross-chain/polygon/deploy_sidechain/31_upgrade_polygon_wormhole_gateway.ts

```diff
@@ -0,0 +1,50 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { ethers, getNamedAccounts, helpers, deployments } = hre
+  const { deployer } = await getNamedAccounts()
+
+  const PolygonTokenBridge = await deployments.get("PolygonTokenBridge")
+  const PolygonWormholeTBTC = await deployments.get("PolygonWormholeTBTC")
+  const PolygonTBTC = await deployments.get("PolygonTBTC")
+
+  const [, proxyDeployment] = await helpers.upgrades.upgradeProxy(
+    "PolygonWormholeGateway",
+    "PolygonWormholeGateway",
+    {
+      contractName:
+        "@keep-network/tbtc-v2/contracts/l2/L2WormholeGateway.sol:L2WormholeGateway",
+      initializerArgs: [
+        PolygonTokenBridge.address,
+        PolygonWormholeTBTC.address,
+        PolygonTBTC.address,
+      ],
+      factoryOpts: { signer: await ethers.getSigner(deployer) },
+      proxyOpts: {
+        kind: "transparent",
+      },
+    }
+  )
+
+  // Contracts can be verified on L2 Polygonscan in a similar way as we do it on
```

Comment:
nitpick
```suggestion
  // Contracts can be verified on Polygonscan in a similar way as we do it on
```

Reply:
not relevant anymore, removed this part.